### ISSUE TITLE
Add pkg extraction support 

### ISSUE THREAD

**Issue by TheOfficialFloW** [TheOfficialFloW/VitaShell#518]

**Comment by KuromeSan** [comment]
Will it be able to promote pkg too?

**Comment by TheRadziu** [comment]
@KuromeSan imo, having issue title in mind, it will ONLY extract the files to wherever user wants, so as long as you have license in your license database[or extracted content is DRM free like CrunchyRoll App] and app/dlc is extracted to proper directory, it should work just fine with existing refresh livearea feature.

**Comment by KuromeSan** [comment]
That's a shame. It'd be kinda cool to be able to use fpkg for homebrew :d

**Comment by FutabaHoneyOTU** [comment]
don't you need the zrif to extract a pkg?

**Comment by TheRadziu** [comment]
@FutabaHoneyOTU no, to extract pkg you need pkg keys, but they are hardcoded in all current pkg extraction tools. zRIF is just a NoNpDRM license to use the title.
